### PR TITLE
dev-tex/pythontex: version bump

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

**Comment by TheChymera** [comment]
@jlec The failure is unrelated to my PR - can you please merge?

**Comment by jlec** [comment]
Same here.

**Comment by jlec** [comment]
@TheChymera please fix the items in the review comments.

**Comment by TheChymera** [comment]
@jlec I see no review comments. Can you confirm you actually submitted any? Link maybe?

**Comment by TheChymera** [comment]
@jlec seriously man, help me out here, there are no review comments for this.

File: dev-tex/pythontex/pythontex-0.15.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{3,4} )
```
Comment (kiwifb) [inline dev-tex/pythontex/pythontex-0.15.ebuild]: python3_3 is off the tree you should put 3_4, 3_5 and possibly 3_6 if you know it is supported.

File: dev-tex/pythontex/pythontex-0.15.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (kiwifb) [inline dev-tex/pythontex/pythontex-0.15.ebuild]: Why not move to the nice EAPI 6?

**Comment by jlec** [comment]
Hi @TheChymera 

I am really sorry. I am using a extension or more precise, I tried to use it. I never actually send the review to you.

**Pull request merged**